src/lib/dvf/token/getSafeQuantizedAmountOrThrow.js

```diff
@@ -3,6 +3,11 @@ const { Long, toQuantizedAmountBN, fromQuantizedToBaseUnitsBN } = require('dvf-u
 
 module.exports = (baseUnitAmount, tokenInfo) => {
   const quantisedAmount = toQuantizedAmountBN(tokenInfo, baseUnitAmount)
+
+  if (quantisedAmount.eq(0)) {
+    return quantisedAmount
+  }
```

Comment:
But isn't quantising 0  = 0 a valid statement? We did something similar in AMM contract's quantiser functions. If the amount was between 0 and quantum then that could be considered risky since its a truncation

Reply:
ok, maybe I misunderstood the purpose of this function

still, not quite sure why truncating 0.4 => 0 should be treated differently than truncating 1.4 = 1